Lib/test/support/__init__.py

```diff
@@ -155,8 +155,12 @@ def get_attribute(obj, name):
     try:
         attribute = getattr(obj, name)
     except AttributeError:
-        raise unittest.SkipTest("module %s has no attribute %s" % (
-            obj.__name__, name))
+        if isinstance(obj, type(sys)):
```

Comment:
This code is a hybrid between former 2.7 code and master code. In 2.7 it can be used only for modules, and reports module name on error. In master it can be used for arbitrary objects and reports object's repr on error (this looks worser for modules). I'm going to port this specialization to master.

`types.ModuleType` is not used because `types` is not imported in `test.support` and I don't want to add imports without need.